surefire-integration-tests/src/test/java/org/apache/maven/surefire/its/fixture/HelperAssertions.java

```diff
@@ -159,4 +164,10 @@ public static IntegrationTestSuiteResults parseReportList( List<ReportTestSuite>
             throw new RuntimeException( "Couldn't parse XML reports", e );
         }
     }
+
+    public static void assumeJavaVersion(JavaVersion version)
+    {
+        assumeThat( "java.specification.version: ",
```

Comment:
Nice PR. Maybe `SystemUtils.JAVA_SPECIFICATION_VERSION + ": ` instead. Do you like static imports?